presto-main/src/main/java/com/facebook/presto/sql/planner/PlanPrinter.java

```diff
@@ -972,6 +981,16 @@ private String formatDomain(Domain domain)
 
             return "[" + Joiner.on(", ").join(parts.build()) + "]";
         }
+
+        private String formatCost(PlanNode node)
+        {
+            PlanNodeCost cost = costs.getOrDefault(node, UNKNOWN_COST);
+            Estimate outputRowCount = cost.getOutputRowCount();
+            Estimate outputSizeInBytes = cost.getOutputSizeInBytes();
+            return String.format("{cost: %s/%s}",
```

Comment:
I think this format will require tribal knowledge to be understood.
`{rows: %s, bytes: %s}`?